parm/soca/varchange/soca_2cice_arctic.yaml

```diff
@@ -16,25 +16,25 @@ variable change:
     min hsno: 0.1
   domain: arctic
   cice background state:
-    restart: $(ICE_RST)
+    restart: {{ ICE_RST }}
     ncat: 5
     ice_lev: 7
     sno_lev: 1
     tstep: PT1H
   cice output:
-    restart: $(ICE_RST)
+    restart: {{ ICE_RST }}
   output variables: [tocn, socn, hocn, cicen, hicen, hsnon]
 
 states:
 - input:
      read_from_file: 1
      basename: ./
-     ocn_filename: $(OCN_ANA)
-     ice_filename: $(ICE_ANA)
-     date: $(FCST_BEGIN)
+     ocn_filename: {{ OCN_ANA }}
```

Comment:
same quote question as above.  Do we need quotes here and around other `{{ }}` which do not yet have quotes?